portfolio/src/main/webapp/js/script.js

```diff
@@ -149,16 +150,29 @@ function checkLogin() {
             commentForm.id = 'show';
             const loggedInAsDiv = document.getElementById('loggedInAs');
             loggedInAsDiv.innerHTML = '<p>You are logged in as ' + data.email + '. Logout <a href=\'' + data.logoutURL + '\'>here</a>.</p>';
+            loggedInAsDiv.innerHTML += '<div id="hidden"><input id="submitted-email" value="'+ data.email +'" name="'+ data.email +'"></div>'
         } else {
             const loginDiv = document.getElementById('login');
             loginDiv.innerHTML = '<p>You are not logged in to leave a comment. Login <a href=\'' + data.loginURL + '\'>here</a>.</p>'
         }
     });
 }
 
+function checkLoginForDeleteAllBtn() {
```

Comment:
Ah, it took me about 10 seconds to realize that "Btn" stands for "Button" :) For naming, the recommendation is "Use the shortest possible name, but not at the cost of clarity. Avoid acronyms, abbreviations, and in-team jargon." Because your code would be read by many other programmers including you after many years. So naming is important for writing good codes.

Reply:
Yes, that was what I was going for, but I suppose I just assumed Btn would be clear, not realizing it wasn't clear to all. I will change this to Button.